cmd/geth/chaincmd.go

```diff
@@ -362,6 +373,39 @@ func exportChain(ctx *cli.Context) error {
 	return nil
 }
 
+// exportHistory exports chain history in Era archives at a specified
+// directory.
+func exportHistory(ctx *cli.Context) error {
+	if ctx.Args().Len() != 3 {
```

Comment:
This looks wrong to me, your checking if args.len == 3, but access 4 elements ("<dir> <first> <last> <step>)

Reply:
Ah right, I added step in later.